### PR TITLE
fix: javascript template strings were not being translated

### PR DESCRIPTION
**Pull request by jansenk**
**TL;DR -** Remove backticked template strings from calls to `gettext` because they were being ignored by translations jobs

JIRA: [AU-606](https://openedx.atlassian.net/browse/AU-606)

**What changed?**

- A very common pain point I would have is when I tried to run translations from my "local" (in virtualenv) ora environment, I would consistently see the process remove the translations for `"ve detected errors on the following tabs: ${tabNames.join("`. Googling led me to find some ... kinda similar issues? Not the same, but similar enough to make me investigate more.  noticed that that was the only backticked text that was showing up in translations at all! There were three others that were just not included.
- I changed them to use normal quotes and concatenation rather than template strings and everything ended up working out nicely.

**Unanswered Questions**
- Why *exactly* was this happening? What caused the difference in behavior?
- Why was the "lms" environment picking up that one mangled instance but not the other (arguably nicer) ones?
- The "lms" xgettext was an older version than my "local" version. was this something purposefully removed?

**Developer Checklist**

- [ ] Reviewed the [release process](https://github.com/edx/edx-ora2/blob/master/.github/release_process.md)
- [ ] Translations and JS/SASS compiled
- [ ] Bumped version number in [setup.py](https://github.com/edx/edx-ora2/blob/a62e81a9b0d89223476967ec3c27f3557a850735/setup.py#L39) and [package.json](https://github.com/edx/edx-ora2/blob/a62e81a9b0d89223476967ec3c27f3557a850735/package.json#L3)

**Reviewer Checklist**

Collectively, these should be completed by reviewers of this PR:

- [ ] I've done a visual code review
- [ ] I've tested the new functionality

FYI: @openedx/content-aurora

### CODE REVIEW

**Review by mattcarter — APPROVED** [review]
Approved. The change in verbiage seems to be innocuous, but let product know.
